libraries/ms-drivers/test/test_drv120_relay.c

```diff
@@ -5,10 +5,11 @@
 #include "ms_test_helpers.h"
 
 // Pin used to enable/disable relay
-const GpioAddress TEST_DRV120_PIN = { GPIO_PORT_A, 8 };
+static GpioAddress s_test_drv120_pin = { GPIO_PORT_A, 8 };
```

Comment:
Minor nit: I think it's ok for this to be `static const` since it's still constant.